cephci/utils/configs.py

```diff
@@ -0,0 +1,99 @@
+import yaml
+
+from cli.exceptions import ConfigError
+
+DEFAULT_CONFIG_PATH = "~/.cephci.yaml"
+CONFIG = None
+
+
+def get_configs(config=DEFAULT_CONFIG_PATH):
+    global CONFIG
+    if CONFIG:
+        return CONFIG
+
+    with open(config, "r") as _stream:
+        try:
+            CONFIG = yaml.safe_load(_stream)
+        except yaml.YAMLError:
+            raise ConfigError(f"Invalid configuration file '{config}'")
+
+
+def _get_credentials():
+    if not CONFIG:
+        raise ConfigError("Configuration is not passed")
+
+    try:
+        return CONFIG["credentials"]
+    except KeyError:
+        raise ConfigError("Credentials configurations are missing from config")
+
+
+def get_subscription_credentials(server):
```

Comment:
one suggestion, the server variable here will have the value `cdn`, `stage` etc right? In that case, can we replace the server with subscription_type or something similar? Please ignore if not valid

Reply:
`cdn`, `stage`  are based on the `serverurl` that's why `server` variable is used here.